src/views/Profile.vue

```diff
@@ -2,7 +2,7 @@
   <div class="profile-wrapper container">
     <div class="columns">
       <div class="column">
-        <div class="columns">
+        <div class="columns content-container">
```

Comment:
"content-container" could be replace by "is-align-items-center"

Reply:
@roiLeo may I know which utility class library we use, so I can take care of this from next time?

Reply:
@prachi00 We use for frontend [Buefy](https://buefy.org/) which is based on [Bulma](https://bulma.io/)